README.md

```diff
@@ -2,16 +2,21 @@
 
 The documentation for the Contao project will be maintained in this repository.
 
+The details below are only necessary if you want to install the docs locally.\
+If you just want to contribute content, read the [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Cloning
 
-The project installs the Hugo Learn theme as a git submodule. Thus when cloning 
-the repository, you need to use the `--recurse-submodules` parameter:
+The project installs the Hugo Learn theme as a git submodule. Thus when cloning the repository, you need to use the `--recurse-submodules` parameter:
```

Comment:
Again, why this change? ;)

Line breaks should happen at or after 80 characters. Since you did not change the content, you shouldn't change anything here.

Reply:
Hm - i thought that this is re(implement) in my patch-2 branch:
https://github.com/fkaminski/docs/tree/patch-2

Reply:
No, you have completely removed the line break after "cloning" now. Just restore the original paragraph.

Reply:
Ok - that way ...